src/truman.js

```diff
@@ -148,7 +148,13 @@ let truman = module.exports = {
             loggingHelper.log(xhr, fixture);
           } else {
             loggingHelper.log(`%cNOT FOUND%c: ${xhr.method} ${xhr.url}`, 'color: red', 'color: black');
-            loggingHelper.log(xhr, fixtures);
+            loggingHelper.log('Looking for XHR:', xhr);
+
+            if (xhr.method === 'GET') {
+              loggingHelper.log('Fixtures (closest URL first):', fixtureHelper.sortByClosestMatchingURL(fixtures, xhr));
```

Comment:
What would that give us? I feel like unless the limiting was smart enough to only limit when the fixtures hit a certain similarity threshold we'd just be creating an edge case where the correct fixture was beyond the list limit.